content/configuration/devtool.md

```diff
@@ -49,6 +49,8 @@ W> There are some issues with Source Maps in Chrome. [We need your help!](https:
 
 `cheap-module-source-map` - A SourceMap without column-mappings that simplifies loaded Source Maps to a single mapping per line.
 
+`nosources` - This option prevents the generation of a SourceMap file for your project.
```

Comment:
I tried this out in my project
 - `devtool: 'nosources'` - No source map files were created.
 - `devtool: false` - No sourcer
 - `devtool : <Any other option>` - Source map file is created

Should it behave differently?